.github/workflows/publish-all-verions.yml

```diff
@@ -22,12 +21,13 @@ jobs:
           pip install -r requirements.txt
 
       - name: Generate HTML
-        run: make html
+        run: |
+          ./buildPublishable.sh mbh-versioning ./publish_build remotes/origin
 
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@v4.3.3
         with:
           token: ${{ secrets.GHPAGES_DEPLOY_TOKEN }}
-          repository-name: openqasm/openqasm.github.io
-          branch: main
-          folder: build/html
+          repository-name: mbhealy/mbhealy.github.io
```

Comment:
I assume this is what you will update before merging?

Reply:
Yes, will be updated before merge.

Reply:
Done in https://github.com/openqasm/openqasm/pull/452/commits/0d676cfb3e7eb5b80ee985a4d5a7aacecc7288e8